_opensearch/rest-api/ingest-apis/simulate-ingest.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: default
+title: Simulate an ingest pipeline
+parent: Ingest APIs
+grand_parent: REST API reference
+nav_order: 13
+---
+
+# Simulate a pipeline
+
+Simulates an ingest pipeline with any example documents set you specify.
+
+## Example
+
+```
+POST /_ingest/pipeline/{id}/_simulate
+{
+  "docs": [
+    {
+      "_index": "index",
+      "_id": "id",
+      "_source": {
+        "location": "document-name"
+      }
+    },
+    {
+      "_index": "index",
+      "_id": "id",
+      "_source": {
+        "location": "document-name"
+      }
+    }
+  ]
+}
+```
+
+## Path and HTTP methods
+
+Simulate the last ingest pipeline created
+
+```
+GET _ingest/pipeline/_simulate
+POST _ingest/pipeline/_simulate
+```
+
+Simulate a single pipeline based on the pipeline's ID.
+
+```
+GET _ingest/pipeline/{id}/_simulate
+POST _ingest/pipeline/{id}/_simulate
+```
+
+## URL parameters
+
+Parameter | Type | Description
+:--- | :--- | :---
+verbose | boolean | Verbose mode. Display data output for each processor in executed pipeline
+
+## Request body fields
+
+Field | Type | Description
+:--- | :--- | :---
+`pipeline` | object | The pipeline you want to simulate. When included without the pipeline `{id}` inside the request path, the response simulates the last pipeline created
+`docs` | array of objects | The documents you want to use to test the pipeline
+
+The `docs` field can include the following subfields:
+
+Field | Type | Description
+:--- | :--- | :---
+`id` (Optional) | string | An optional identifier for the document. Cannot be used elsewhere in the index
+`index` (Optional) | string |The index where the documents transformed data will be stored
```

Comment:
document's? or are you talking about multiple documents. in that case, maybe "the transformed data from the documents will be stored" or something.